doc/README.md

```diff
@@ -136,9 +136,15 @@ Panning is special because it's accessible in any mode (if you have a keyboard)
 by holding down the <kbd>Space</kbd> key and then dragging the page.
 (This behavior matches Adobe Creative Suite.)
 
+A mouse scroll-wheel or a touchpad's scroll behavior (typically
+two-finger dragging) will also pan the canvas.
+
 Related, if you want to zoom into or out of the page, check out the
 [<img src="icons/search-minus.svg" width="18" alt="Zoom Out Icon"> Zoom Out /
-<img src="icons/search-plus.svg" width="18" alt="Zoom In Icon"> Zoom In buttons](#-zoom-out---zoom-in).
+<img src="icons/search-plus.svg" width="18" alt="Zoom In Icon"> Zoom In buttons](#-zoom-out---zoom-in),
+which can also be activated by using a mouse scroll-wheel while
```

Comment:
Maybe make this start a new sentence, like "You can also zoom by using ..." (also remove hyphen in "scroll-wheel")

Reply:
Implemented